README.aspnet.md

```diff
@@ -1,3 +1,9 @@
+## Important: Client Firewall Rules Update to Microsoft Container Registry (MCR)
+
+To provide a consistent FQDNs, on **March 3, 2020** the data endpoint will be changing from *.cdn.mscr.io to *.data.microsoft.com. 
+
+For more info, see [MCR Client Firewall Rules](https://aka.ms/mcr/firewallrules).
+---------------------------------------------------------------------------------
```

Comment:
There should be a blank line before the next header.

Reply:
fixed